Readme.md

```diff
@@ -7,7 +7,7 @@
 ## Example
 
 ```js
-var snippet = require('snippet');
+var snippet = require('segmentio-snippet');
 
 var contents = snippet.max({
   host: 'cdn.segment.com',
```

Comment:
Can you update the snippet with supported platforms? This change kind of changes the purpose of this package from mostly internal to customer-facing; would be good to message the intent of the package.

I think we should also include messaging around using this not being recommended practice, instead pointing customers toward usage of the snippet in the head. Encouraging usage of this package is going to have a success/support impact related to `window.analytics` not always being defined the way it would be per our installation instructions.